josm-presets/de.xml

```diff
@@ -1740,16 +1740,52 @@ See http://wiki.openstreetmap.org/wiki/OpenRailwayMap for details.
 				/>
 		</item>
 		<separator />
-		<item name="Grenzpunkt" icon="owner-change.png" type="node">
-			<label text="Eigentumswechsel der Infrastruktur an Privatbahnanschlüssen und Landesgrenzen" />
+		<item name="Change of Operator" de.name="Betreiberwechselpunkt" icon="owner-change.svg" type="node">
```

Comment:
The term "Betreiberwechselpunkt" sounds a bit strange, I have never heard of this. I would propose the terms "Betreiberwechsel", "Betreibergrenze" or "Anschlussgrenze" instead.